src/hotspot/share/opto/phaseX.cpp

```diff
@@ -1594,13 +1594,14 @@ void PhaseIterGVN::add_users_to_worklist( Node *n ) {
       }
     }
 
-    // If changed Cast input, notify down for Phi and Sub - both do "uncast"
+    // If changed Cast input, notify down for Phi, Sub, and Xor - all do "uncast"
     // Patterns:
     // ConstraintCast+ -> Sub
     // ConstraintCast+ -> Phi
+    // ConstraintCast+ -> Xor
     if (use->is_ConstraintCast()) {
       auto push_phi_or_sub_uses_to_worklist = [&](Node* n){
```

Comment:
```suggestion
      auto push_the_uses_to_worklist = [&](Node* n){
```

Reply:
Change the name now. Maybe something more generic is good, since we do not know how many more of these examples we will eventually add to this.